spec/codebreaker_console/test_users.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Move this file to `spec/support/` folder

Reply:
moved